src/Dialogs/Plane/PlanePolarDialog.cpp

```diff
@@ -187,8 +189,9 @@ PlanePolarWidget::ListClicked()
 
   const PolarStore::Item &item = PolarStore::GetItem(list[result].int_value);
 
-  plane.reference_mass = item.reference_mass;
-  plane.dry_mass = item.reference_mass;
+  plane.polar_shape.reference_mass = item.reference_mass;
+  auto crew_mass_estimate = StaticString<32>(item.name).Contains(_T("PAS)")) ? 180. : 90.;
+  plane.empty_mass = item.reference_mass - crew_mass_estimate; // todo needs rework on polar store data
```

Comment:
Can we solve this todo before merging this PR?

Reply:
Yes, no way around that. The commit is according to me UML sketch, but not complete. Just going forward not having a buy in would be a mess.

Reply:
Removed this logic with the intention to commit the reorganized polar store in a second commit incl. the proper empty mass default..